From 292c60b91bd0a6e7198261e4a586d711dd390bb3 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Wed, 22 Jan 2020 22:48:40 +0100
Subject: fix build

m_bStartGenericField became m_nStartGenericField, but more importantly a
closing bracket was wrongly added (well, left over)  during merge.

regression from a290d025339ef92b93ba6d287658ba784a818220

Change-Id: Ic686e9c95fa9e5a95cf3264d4484e917dcbd3c98
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/87225
Reviewed-by: Thorsten Behrens <Thorsten.Behrens@CIB.de>
Tested-by: Thorsten Behrens <Thorsten.Behrens@CIB.de>
---
 writerfilter/source/dmapper/DomainMapper_Impl.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/writerfilter/source/dmapper/DomainMapper_Impl.cxx b/writerfilter/source/dmapper/DomainMapper_Impl.cxx
index be95eba34308..8f6ca68470e5 100644
--- a/writerfilter/source/dmapper/DomainMapper_Impl.cxx
+++ b/writerfilter/source/dmapper/DomainMapper_Impl.cxx
@@ -1878,7 +1878,7 @@ void DomainMapper_Impl::appendTextPortion( const OUString& rString, const Proper
                         uno::Reference< text::XTextCursor > xTOCTextCursor = xTextAppend->getEnd()->getText( )->createTextCursor( );
                         assert(xTOCTextCursor.is());
                         xTOCTextCursor->gotoEnd(false);
-                        if (m_bStartIndex || m_bStartBibliography || m_bStartGenericField != 0)
+                        if (m_bStartIndex || m_bStartBibliography || m_nStartGenericField != 0) {
                             xTOCTextCursor->goLeft(1, false);
                         }
                         xTextRange = xTextAppend->insertTextPortion(rString, aValues, xTOCTextCursor);
-- 
cgit v1.2.1

